rotkehlchen/api/rest.py

```diff
@@ -4274,17 +4278,22 @@ def get_evm_products(self) -> Response:
     @async_api_call()
     def refresh_general_cache(self) -> dict[str, Any]:
         eth_node_inquirer = self.rotkehlchen.chains_aggregator.ethereum.node_inquirer
-        curve_success = eth_node_inquirer.ensure_cache_data_is_updated(
-            cache_type=CacheType.CURVE_LP_TOKENS,
-            query_method=query_curve_data,
-            save_method=save_curve_data_to_cache,
-            force_refresh=True,
-        )
-        if curve_success is False:
-            return wrap_in_fail_result(
-                message='Failed to refresh curve pools cache',
-                status_code=HTTPStatus.CONFLICT,
-            )
+        eth_caches = {
+            'curve pools': (CacheType.CURVE_LP_TOKENS, query_curve_data, save_curve_data_to_cache),
+            'convex pools': (CacheType.CONVEX_POOL_ADDRESS, query_convex_data, save_convex_data_to_cache),  # noqa: E501
+        }
```

Comment:
does it make sense for it to be a dict? Why not make it a tuple?

```suggestion
        eth_caches = (
            ('curve pools', CacheType.CURVE_LP_TOKENS, query_curve_data, save_curve_data_to_cache),
            ('convex pools', CacheType.CONVEX_POOL_ADDRESS, query_convex_data, save_convex_data_to_cache),  # noqa: E501
        )
```